_data/compatibility/binance.yaml

```diff
@@ -0,0 +1,67 @@
+---
+name: Binance
+internal_url: /en/compatibility/binance
+logo: /img/compatibility/binance/binance.png
+rbf:
+  tested:
+    date: "2018-11-06"
+    platforms:
+      - web
+    version: "n/a"
+  features:
+    receive:
+      notification: "false"
+      list: "false"
+      details: "false"
+      shows_replaced_version: "false"
+      shows_original_version: "false"
+    send:
+      signals_bip125: "false"
+      list: "false"
+      details: "false"
+      shows_replaced_version: "na"
+      shows_original_version: "na"
+  examples:
+    - image: /img/compatibility/binance/send-screen.png
+      caption: >
+        Sending RBF Transaction - Default send (withdrawal) screen. No RBF options.
+    - image: /img/compatibility/binance/transaction-list-sent.png
+      caption: >
+        Bumping RBF Transactiont - Transaction not sent via RBF. No fee bump options.
+    - image: /img/compatibility/binance/transaction-list-incoming-rbf.png
+      caption: >
+        Receiving RBF Transaction - No unconfirmed transactions appear in transaction list.
+    - image: /img/compatibility/binance/transaction-list-replacement-confirmed.png
+      caption: >
+        Receiving Bumped RBF Transaction - Replacement transaction only shows up after confirmed. Neither transaction shows before.
+segwit:
+  tested:
+    date: "2019-04-11"
+    platforms:
+      - web
+    version: "n/a"
+  features:
+    receive:
+      p2sh_wrapped: "false"
+      bech32: "false"
+      default: "p2pkh"
+    send:
+      bech32: "true" 
+      change_bech32: "untested"
+      segwit_v1: "Does not pass validation"
```

Comment:
I don't know what "Does not pass validation" means.  Since this is being passed to the wallet details page as text, I think it should be more descriptive about the problem and possibly about what the wallet author should do to fix the problem.

Reply:
You brought up the valid concern of whether we should even show segwit v1 results. Im going to leave this for now until we decide on that.